polaris-react/src/components/Page/components/Header/Header.scss

```diff
@@ -15,6 +15,29 @@ $action-menu-rollup-computed-width: 40px;
 
 .BreadcrumbWrapper {
   grid-area: breadcrumbs;
+
+  #{$se23} & {
+    /* stylelint-disable-next-line selector-max-combinators -- SE23 match button group */
+    a,
+    button {
+      background-color: var(--p-color-bg-strong);
+      border-radius: var(--p-border-radius-2);
+      box-shadow: none;
+
+      /* stylelint-disable-next-line selector-max-combinators -- SE23 match button group */
+      &:hover {
+        box-shadow: none;
+        background-color: var(--p-color-bg-strong-hover);
+      }
+
+      /* stylelint-disable-next-line selector-max-combinators -- SE23 match button group */
+      &:focus {
+        background-color: var(--p-color-bg-strong-active);
+        /* stylelint-disable-next-line declaration-no-important -- SE23 match button group */
+        box-shadow: var(--p-shadow-inset-md) !important;
```

Comment:
I think I need `!important` here because the focus state for `Button` is highly specific. This is also done in the other buttons

Reply:
Not ideal, but I think this solution could be refactored more systematically across the other areas as well later on.

For now, this is a "patterned fix" for this type of issue 👍